amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -1790,6 +1790,33 @@ VX_API_ENTRY vx_node VX_API_CALL vxExtrppNode_ResizeCropMirrorPD(vx_graph graph,
     return node;
 }
 
+VX_API_ENTRY vx_node VX_API_CALL vxExtrppNode_ResizeMirrorNormalizebatchPD(vx_graph graph, vx_image pSrc, vx_array srcImgWidth, vx_array srcImgHeight,vx_image pDst,vx_array dstImgWidth,vx_array dstImgHeight,vx_array mean, vx_array std_dev, vx_array flip, vx_scalar chnShift, vx_uint32 nbatchSize)
+{
+	vx_node node = NULL;
+	vx_context context = vxGetContext((vx_reference)graph);
+	if(vxGetStatus((vx_reference)context) == VX_SUCCESS) {
+		vx_uint32 dev_type = getGraphAffinity(graph);
+		vx_scalar DEV_TYPE = vxCreateScalar(vxGetContext((vx_reference)graph), VX_TYPE_UINT32, &dev_type);
+		vx_scalar NBATCHSIZE = vxCreateScalar(vxGetContext((vx_reference)graph), VX_TYPE_UINT32, &nbatchSize);
+		vx_reference params[] = {
+			(vx_reference) pSrc,
+            (vx_reference) srcImgWidth,
```

Comment:
Please align with other params

Reply:
Some are in tab and some in space. Convert everything to tab/space.